scipy/stats/_morestats.py

```diff
@@ -2841,7 +2842,7 @@ def ansari(x, y, alternative='two-sided'):
 
 
 @_axis_nan_policy_factory(BartlettResult, n_samples=None)
-def bartlett(*samples):
+def bartlett(*samples, axis=0):
```

Comment:
Should be keyword only no?

Reply:
Yes. It is.

Reply:
Ah yes 👍 always forget the distinction with the splat for any number of positional args. Sorry for the noise